src/snyk/common/watchers/configurationWatcher.ts

```diff
@@ -25,6 +27,9 @@ class ConfigurationWatcher implements IWatcher {
   constructor(private readonly logger: ILog) {}
 
   private async onChangeConfiguration(extension: IExtension, key: string): Promise<void> {
+    if (key === ADVANCED_ORGANIZATION_SETTING) {
+      await configuration.fetchAndSetFeatureFlag(FEATURE_FLAGS.consistentIgnores);
```

Comment:
Could we not make this call for `ADVANCED_ORGANIZATION` so we don't need to add a new configuration key? We basically want to update the feature flag whenever the org changes right?

Reply:
This check is for the settings page. 

Is there another way to listen for a organisation change?

![image](https://github.com/snyk/vscode-extension/assets/1948377/6857657e-95d3-4da3-93bd-8fa657b6adc2)

Reply:
I'm saying that I **think** `ADVANCED_ORGANIZATION` is already meant to represent a change for that panel: https://github.com/snyk/vscode-extension/blob/main/src/snyk/common/constants/settings.ts#L19. It looks like we're not listening to this in `onChangeConfiguration` though, for some weird reason.

What is the difference between `ADVANCED_ORGANIZATION_SETTING` and `ADVANCED_ORGANIZATION`?

Reply:
Oh! I'm sorry! You were saying that I duplicated a variable!
You are right! I didn't realise I duplicated `snyk-advanced.organization`.
There is no difference, I made a mistake declaring a variable that already exists. 
Deleting it now, thank you.